Gui/opensim/nbproject/project.properties.in

```diff
@@ -0,0 +1,48 @@
+app.icon=branding/core/core.jar/org/netbeans/core/startup/frame48.gif
+app.icon.icns=installer_files/OSX/opensim.icns
+app.name=OpenSim
+app.title=OpenSim
+app.version=OpenSim-4.0-@VERSION@
```

Comment:
My preference is for this to simply be `@VERSION@` and for the `4.0-` to come from CMake. That way, for the tagged released, we can have `@VERSION@` be only `4.0` (with no commit hash or date).